src/content/app/species-selector/state/species-selector-general-slice/speciesSelectorGeneralSlice.ts

```diff
@@ -142,6 +145,9 @@ const speciesSelectorGeneralSlice = createSlice({
   reducers: {
     updateCommittedSpecies(state, action: PayloadAction<CommittedItem[]>) {
       state.committedItems = action.payload;
+    },
+    setLozengeDisplaySelection(state, action: PayloadAction<string>) {
```

Comment:
Names again. I've mentioned in a comment to SpeciesLozengeDisplaySelector how we shouldn't really keep the word lozenge when referring to this setting; but here you also have the word "selection", which is another ui word. `setSpeciesNameDisplayOption`?